Projects/Feature/BottleStorage/Interface/Sources/PingPongDetail/View/PingPongDetail/PingPongDetailFeatureInterface.swift

```diff
@@ -29,7 +29,14 @@ public struct PingPongDetailFeature {
     var isStopped: Bool {
       return pingPong?.isStopped == true || pingPong?.isStopped == nil
     }
-    
+    var isMatchingTabDisabled: Bool {
+      guard let matchStatus = pingPong?.matchResult.matchStatus
+      else {
+        return false
+      }
```

Comment:
```suggestion
      guard let matchStatus = pingPong?.matchResult.matchStatus else { return false }
```

```suggestion
      guard let matchStatus = pingPong?.matchResult.matchStatus else { 
        return false
       }
```

이거 형 코드 사용하는거보니깐 else를